integration-test/546-road-sort-keys-bridges.py

```diff
@@ -13,7 +13,7 @@ def test_motorway_bridge(self):
         self.assert_has_feature(
             16, 10472, 25323, 'roads',
             {'kind': 'highway', 'kind_detail': 'motorway', 'id': 28412298,
-             'name': 'Presidio Pkwy.', 'is_bridge': True, 'sort_rank': 443})
+             'name': type(None), 'is_bridge': True, 'sort_rank': 443})
```

Comment:
This is because the feature is too short to fit the label, while at zoom 17 it would fit / we don't drop.